docs/semgrep-supply-chain/sbom.md

```diff
@@ -0,0 +1,52 @@
+---
+slug: /semgrep-supply-chain/sbom
+append_help_link: true
+title: SBOM
+hide_title: true
+description: Generate a CycloneDX JSON or XML SBOM to view all dependencies of a repository.
+tags:
+  - Semgrep Supply Chain
+---
+
+# Generating an SBOM (software bill of materials)
+
+Generate a software bill of materials (SBOM) to assess your third-party dependencies and comply with auditing procedures. Semgrep Supply Chain (SSC) can generate an SBOM for each repository you have added to Semgrep Cloud Platform.
+
+## Supported standards and formats
+
+Semgrep Supply Chain supports the following:
+
+- CycloneDX 1.4 JSON
+- CycloneDX 1.4 XML
+
+## Generating and downloading an SBOM for a single project
+
+:::info Prerequisites
+- **SBOM generation** can be performed only through Semgrep Cloud Platform (SCP). [Create an account](/semgrep-code/getting-started/#signing-in-to-semgrep-cloud-platform) to use this feature.
+- You need at least 1 successful Supply Chain scan on the **trunk branch** of each repository you want to generate an SBOM for. To add a repository to Semgrep for scanning, see [<i class="fa-regular fa-file-lines"></i> Running scans](/semgrep-cloud-platform/getting-started/#running-scans).
+:::
+
+1. In Semgrep Cloud Platform, click **Supply Chain > Dependencies**. 
```

Comment:
going to dependencies tab means you can download SBOM per project AFAIK. 
If the user wants to download SBOM for *all* the projects, then should we ask them to go the to the "Vulnerabilities" tab and hit the SBOM button?
cc: @andy-r2c

Reply:
the "All projects" feature doesn't work yet :)